templates/templated_email/html/source/dashboard/staff/promote_customer.mjml

```diff
@@ -0,0 +1,30 @@
+<mjml>
+  <mj-head>
+    <mj-include path="../../shared/styles" />
+  </mj-head>
+  <mj-body>
+    <mj-container>
+      <mj-section css-class="no-display">
+        <mj-column>
+          <mj-text>
+            {% load i18n %}
+          </mj-text>
+        </mj-column>
+      </mj-section>
+      <mj-include path="../../shared/header" />
+      <mj-section>
+        <mj-column>
+          <mj-text  font-size="16px">
```

Comment:
Seems like extra space left after attr's deletion? This repeats everywhere that `<mj-text  font-size="16px">` was copied to.

Reply:
You're right, I'll fix it.